modules/nw-metallb-operator-setting-runtimeclass.adoc

```diff
@@ -47,12 +47,12 @@ spec:
     runtimeClassName: myclass
     annotations: <1>
       controller: demo
-  speakerConfig: 
+  speakerConfig:
     runtimeClassName: myclass
     annotations: <1>
       speaker: demo
 ----
-<1> This example uses `annotations` to add metadata such as build release information or GitHub pull request information. You can populate annotations with characters not permitted in labels. However, you cannot use annotations to identify or select objects. 
+<1> The earlier example uses `annotations` to add metadata such as build release information or GitHub pull request information. You can populate annotations with characters that are no permitted in labels. However, you cannot use annotations to identify or select objects.
```

Comment:
Is this supposed to be "not"? Also, I am confused with the use of "The earlier example" does that mean the example from #1 or does it mean the example this callout is in? If it is this example, I would keep the "This"

```suggestion
<1> The earlier example uses `annotations` to add metadata such as build release information or GitHub pull request information. You can populate annotations with characters that are not permitted in labels. However, you cannot use annotations to identify or select objects.
```

Reply:
hi @opayne1 I may have inadvertently touched that I think I was asked to add "previous" example but vale recommended use of "earlier"

Reply:
To be honest I really think that description is not needed at all but I will revert to original here and I do need to revise this section in a separate PR, there is something else that needs clarification too IMO.